### PR TITLE
Improved Introduction to duecredit

### PR DESCRIPTION
**Pull request by raamana**
Hey guys!

Good job on gettting started in trying to solve this important problem. I appreciate the motivation and support the ideas. 

But the README was difficult for me to follow, so I tried to improve it. I'm not sure if I even understand it correctly, this is to try improve it for everyone. Please feel free to correct and expand it as you see it.

I will add this to my software once I get a handle on it.

Pradeep

### CODE REVIEW

**Comment by raamana** [comment]
In the next revision, I will add a link to you [slides](https://github.com/duecredit/duecredit-publications/blob/master/talks/neurohackweek2016/duecredit.pdf), which are much easier to follow.

File: README.md [hunk 1]
```diff
@@ -15,18 +15,37 @@ level of reference detail, i.e. only references for actually used
...
+### To cite the modules you are using 
 
 You can already start "registering" citations using duecredit in your
 Python modules and even registering citations (we call this approach "injections")
 for modules which do not (yet) use duecredit.  duecredit will remain an optional
 dependency, i.e. your software will work correctly even without duecredit installed.
 
-"Native" use of duecredit (recommended)
----------------------------------------
+For example, list of references to the modules `yourproject` uses ca be produced with few simple commands:
```
Comment (mvdoc) [inline README.md]: How about

```
For example, you can list the citations of methods that `yourproject` uses with few simple commands
```

File: README.md [hunk 2]
```diff
@@ -15,18 +15,37 @@ level of reference detail, i.e. only references for actually used
...
+
+Duecredit is easy to install via pip, simply type:
+ 
+ `pip install duecredit`
+
+## Examples
 
-duecredit 101
-=============
+### To cite the modules you are using
```
Comment (mvdoc) [inline README.md]: it's more methods than modules, because we can wrap also individual functions...how about `modules --> methods`?

**Comment by mvdoc** [comment]
Thanks so much for this, Pradeep! Indeed the README felt a bit of a braindump between us ;) An opinion from outside was much needed. 

I just added some comments about the use of the term `modules`.

File: README.md (hunk 1 above)
Comment (raamana) [inline README.md]: Sounds good. Feel free to make changes! Let me know if I have commit again (I don't think so, right?)

File: README.md [hunk 3]
```diff
@@ -258,8 +281,7 @@ standardization of scientific software development, thus addressing
 many (if not all) core problems with scientific software development
 everyone likes to bash about (reproducibility, longevity, etc.).
```
Comment (raamana) [inline README.md]: Also, I think section needs to expanded further on explaining the problem ( I get the gist of gist). We can add more details backing up this claim (citations or blog posts etc). If we can make this claim stronger, we convince the reader to take this up.

File: README.md (hunk 2 above)
Comment (raamana) [inline README.md]: Sounds good.

File: README.md [hunk 4]
```diff
@@ -234,11 +258,10 @@ that for consistency across projects, you would use following tags
...
-==============
+## Ultimate goals
+
 
-Reduce demand for prima ballerina projects
-------------------------------------------
+### Reduce demand for prima ballerina projects
 
 **Problem**: Scientific software is often developed to gain citations for
 original publication through the use of the software implementing it.
```
Comment (raamana) [inline README.md]: Also, I think section needs to expanded further on explaining the problem ( I get the gist of gist). We can add more details backing up this claim (citations or blog posts etc). If we can make this claim stronger, we convince the reader to take this up.

File: README.md (hunk 2 above)
Comment (yarikoptic) [reply README.md]: I would say modules and methods

**Comment by raamana** [comment]
made more changes based on both of your suggestions.

**Pull request merged**